staging/src/k8s.io/component-base/logs/datapol/externaltypes.go

```diff
@@ -34,8 +35,7 @@ func GlobalDatapolicyMapping(v interface{}) []string {
 }
 
 func byType(t reflect.Type) []string {
```

Comment:
arguably we should just inline this, the compiler probably does, and we only have one call site (including tests) for this method